services/qingstor/errors.go

```diff
@@ -7,4 +7,6 @@ import (
 var (
 	// ErrInvalidBucketName will be returned while bucket name is invalid.
 	ErrInvalidBucketName = errors.New("invalid bucket name")
+	// ErrInvalidWorkDir will be returned while work dir is invalid.
+	ErrInvalidWorkDir = errors.New("invalid work dir, must start and end with only one '/'")
```

Comment:
`invalid work dir` is enough, move `start and end with only one '/'` to comments and docs.